sktime/forecasting/base/adapters/_pytorch.py

```diff
@@ -66,36 +71,37 @@ def _fit(self, y, fh, X=None):
             self._fh = fh
 
         self._y = y
-
-        if type(fh) is ForecastingHorizon:
-            self.network = self._build_network(fh._values[-1])
+        if isinstance(self.fh, ForecastingHorizon):
```

Comment:
seems like a superfluous check, in `_fit`, the `fh` is guaranteed `ForecastingHorizon`

Reply:
You are probably right, I just replaced the old code with isinstance to be more pythonic. I will check if this is needed.

Reply:
I removed that check